common/transport/amqp/test/_amqp_test.js

```diff
@@ -574,7 +574,7 @@ describe('Amqp', function () {
           });
         });
 
-        it('calls the done callback with an error if the connection fails while trying to attach the link', function(testCallback) {
+        it.only('calls the done callback with an error if the connection fails while trying to attach the link', function(testCallback) {
```

Comment:
>it.only [](start = 8, length = 7)

left over from a debugging session?  I don't think you mean to disable all the other tests in the file.

Reply:
correct, thanks for catching that.

---
In reply to: [145962077](https://github.com/Azure/azure-iot-sdk-node/pull/114#discussion_r145962077) [](ancestors = 145962077)